.murdock

```diff
@@ -290,7 +290,7 @@ _greplist() {
 # get list of all app directories
 get_apps() {
     make -f makefiles/app_dirs.inc.mk info-applications \
-        | $(_greplist $APPS) | sort
+        | $(_greplist $APPS) | $(_greplist tests/driver_enc28j60 tests/driver_encx24j600 tests/driver_dose tests/driver_ethos tests/driver_w5100 tests/driver_esp_eth tests/driver_stm32_eth tests/driver_sam0_eth tests/driver_netdev_common) | sort
```

Comment:
(for the next time: just uncomment and edit `APPS` at the top of the file.)

Reply:
Thanks